netci.groovy

```diff
@@ -15,11 +15,16 @@ def static getBuildJobName(def configuration, def os) {
 }
 
 [true, false].each { isPullRequest ->
-    ['Debug', 'Release'].each { configuration ->
+    ['Debug', 'ci_part1', 'ci_part2'].each { configuration ->
```

Comment:
And now I have discovered that public sign and ngen don't work together ... which means that we jit the compiler dll's and fsharp core for every test.

I will see if I can make use of the hosted compiler that we use for the fsharpqa tests.